specification/automanage/resource-manager/Microsoft.Automanage/preview/2021-04-30-preview/automanage.json

```diff
@@ -1283,6 +1283,55 @@
           "application/json"
         ]
       }
+    },
```

Comment:
>}, [](http://example.com/codeflow?start=4&length=2)

Are you planning to implement this as a singleton ?

You would have to define the name since there is only a GET otherwise the customer cannot call it 

Generally people use default or current as singleton name
This is explained here : 
[https://github.com/Azure/azure-resource-manager-rpc/blob/f1fd954cebb3958a2a2ccb62b949bf60491866e0/v1.0/common-api-contracts.md#singleton-resources](https://github.com/Azure/azure-resource-manager-rpc/blob/f1fd954cebb3958a2a2ccb62b949bf60491866e0/v1.0/common-api-contracts.md#singleton-resources) 

Also since this is now an object , you must support the collection get as explained above

Reply:
Thank you Suhas! I will fix it next week

Reply:
Fixed!